stage1/prepare-app/prepare-app.c

```diff
@@ -230,6 +230,47 @@ static void mount_sys(const char *root)
 	mount_at(root, &sys_bind);
 }
 
+static void copy_volume_symlinks()
+{
+	DIR *volumes_dir;
+	struct dirent *de;
+	const char *rkt_volume_links_path = "/rkt/volumes";
+	const char *dev_rkt_path = "/dev/.rkt";
+
+	pexit_if(mkdir(dev_rkt_path, 0700) == -1 && errno != EEXIST,
+		"Failed to create directory \"%s\"", dev_rkt_path);
+
+	pexit_if((volumes_dir = opendir(rkt_volume_links_path)) == NULL && errno != ENOENT,
```

Comment:
I could return if `errno == ENOENT` instead of not failing and relying on `volumes_dir` to be `NULL` but I think this is fine.